pom.xml

```diff
@@ -28,7 +28,9 @@
                         <target>1.8</target>
                         <compilerArgs>
                             <arg>-parameters</arg>
-                            <arg>-Xlint:all,-options,-path</arg>
+                            <arg>-Xlint:unchecked</arg>
+                            <arg>-Xlint:deprecation</arg>
+                            <arg>-Werror</arg>
```

Comment:
I made this change to keep the `compilerArgs` [consistent](https://github.com/playframework/play-meta/issues/13) across sample apps. Not very opinionated about it.